### ISSUE TITLE
Error running custom tool RazorGenerator in cshtml file in .net standard console project in VS 2017

### ISSUE THREAD

**Issue by gabomgp** [aspnet/Tooling#1004]
When i'm trying to run as custom tool the plugin Razor Generator (https://github.com/RazorGenerator/RazorGenerator) in Visual Studio 2017 in a cshtml file, within a .net standard console project, i'm getting the error message "Error to save the edited properties enumerated to continuation. Custom Tool. One or more values are not valid. Internal error."

**Comment by atnegrete** [comment]
Hi, where you ever able to fix this? I'm having a similar problem.

**Comment by gabomgp** [comment]
Yeah. The best "solution" i found is to install the package RazorGenerator.MsBuild and don configure the custom tool. The project compile without problems and in compilation time, the cshtml is transpiled to cs. But, in the error window in Visual Studio, appears errors that dont affects compilation. It's all.

**Comment by barrytang** [comment]
@alexgav @NTaylorMullen @rynowak

**Comment by barrytang** [comment]
To report new issues, use the [Report a Problem](https://developercommunity.visualstudio.com/) tool. The GitHub Issue Tracker for the aspnet/Tooling repo is now deprecated in favor of the Report a Problem tool.

If you encounter a problem with Visual Studio, we want to know about it so that we can diagnose and fix it. By using the Report a Problem tool, you can collect detailed information about the problem, and send it to Microsoft with just a few button clicks. See [here](https://docs.microsoft.com/en-us/visualstudio/ide/talk-to-us) for more details.

**Comment by NTaylorMullen** [comment]
I'd recommend logging an issue over at the RazorGenerator repo. Sadly it's not an officially supported project, some team members just developed it in their spare time 😄.

If you'd like to invest in building your own tool to do things similar to what RazorGenerator did I'd recommend looking at https://github.com/aspnet/Razor/tree/dev/src/RazorPageGenerator.
